### PR TITLE
[unit-tests-only] Add unit tests for SkipTo component

### PR DESCRIPTION
**Pull request by saw-jan**
## Description
Adds unit  tests for `SkipTo` component

## Related Issue
- Part of #5234

## Motivation and Context
- increase unit tests coverage

## How Has This Been Tested?
- test environment: local & CI

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:
- [ ] Code changes
- [x] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: packages/web-runtime/tests/unit/components/SkipTo.spec.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { shallowMount } from '@vue/test-utils'
+import SkipTo from '../../../src/components/SkipTo.vue'
```
Comment (kiranparajuli589) [inline packages/web-runtime/tests/unit/components/SkipTo.spec.js:2]:
```suggestion
import SkipTo from '@files/src/components/SkipTo.vue'
```

**Review by kiranparajuli589 — COMMENTED** [review]
lgtm 👍🏼

File: packages/web-runtime/tests/unit/components/SkipTo.spec.js (hunk 1 above)
Comment (saw-jan) [reply packages/web-runtime/tests/unit/components/SkipTo.spec.js]: it should be `@runtime` but it has not been mapped.

**Review by pascalwengerter — APPROVED** [review]
LGTM

**Pull request merged**